### ISSUE TITLE
.nomedia have no effects inside season folder

### ISSUE THREAD

**Issue by kien-truong** [xbmc/xbmc#23975]
### Describe the bug
For TV shows, files inside seasons folders are still included in library, even when there's a `.nomedia` file in that season folder.

## Expected Behavior
It should be possible to exclude entire seasons from scanning by putting a `.nomedia` file in that season folder.

## Actual Behavior
File inside season folders are not being excluded.

### To Reproduce
1. Setup a tv show using season folders like in the wiki: https://kodi.wiki/view/Naming_video_files/Seasons
2. Put a `.nomedia` file inside a season
3. Rescan library, files inside the season with `.nomedia` is still included.

### Debuglog

The debuglog can be found here:

### Screenshots
Here are some links or screenshots to help explain the problem:

## Additional context or screenshots (if appropriate)
Here is some additional context or explanation that might help:

### Your Environment
Used Operating system:

 - [x] Android
 - [ ] iOS
 - [ ] tvOS
 - [ ] Linux
 - [ ] OSX
 - [ ] Windows
 - [ ] Windows UWP

 - Operating system version/name: Android TV 10/11
 - Kodi version: 20.2

*note: Once the issue is made we require you to update it with new information or Kodi versions should that be required.
Team Kodi will consider your problem report however, we will not make any promises the problem will be solved.*

**Comment by mihawk90** [comment]
I just ran across a similar issue on Xbox.
Inside some of my Anime Folders I have a `_files` directory in which I keep the original filename, and then I have season folders where I keep the same files (hardlinked) using scrapable filenames.
I put a `.nomedia` file inside `_files` and scanned, and yet all the files inside were scanned into the library, of course everything mismatched.

Oddly, I also went into the video sources and specifically set the "content" of `_files` to `None` and even enabled the "Exclude from scan" option. I confirmed the dialogue to delete all database entries in that path and the entire Show was gone. But upon re-scan, it scanned everything back in including `_files` which should have been excluded with 3 separate methods.
